console/src/components/Services/Data/TableBrowseRows/localStorageUtils.ts

```diff
@@ -0,0 +1,95 @@
+const setLSState = (key: string, data: any) => {
+  window.localStorage.setItem(key, JSON.stringify(data));
+};
+
+const getLSState = (key: string, defaultValue: null | {} = {}) => {
```

Comment:
Does this assume only JSON type values?
In that case, we must call the function something else. Maybe `getJSONFromLS` or something? Same for `setLSState`.

I'd rather prefer `getLSState` function that just returns a string and another util function that takes a string and parses it into whatever you want it to parse as.